docs/SRS-Volere/SRS.tex

```diff
@@ -54,9 +54,23 @@ \section*{Revision History}
 ~\newpage
 \section{Purpose of the Project}
 \subsection{User Business}
-\lips
+
+With the world currently facing record high inflation, cost-of-living is at the highest
+it has ever been. This affects all daily necessities but is especially true for food and groceries.
+As a whole, all households are affected but there is particular financial strain on those with lower-incomes.
+As a result, interest in personal finance has grown and become more important in peoples' everyday lives.
+To assist these individuals, we are developing an application that can help users better understand their
+spending habits and make smarter financial decisions. This application will allow users to
+take photos of grocery receipts and track their overall spending, analyze spending trends, and receive
+suggestions on cheaper alternatives for purchased grocery items. Overall, we believe this application
+will help users stay more informed and reduce grocery spending in the long-term.
+
 \subsection{Goals of the Project}
-\lips
+\begin{itemize}
+  \item The created application will help users save money on groceries over time.
+  \item The application will provide accurate spending data and suggestions to end users.
+  \item The application will be accessible and easy-to-use for a large number of people.
```

Comment:
This is more requirement than goal i think

Reply:
i agree with this last one

Reply:
Okay, I removed that last goal.